sdk/react/panels/cast/CastDevicesScreen.js

```diff
@@ -116,10 +127,12 @@ class CastDevicesScreen extends Component {
             marginTop: 20,
           }]}
         >
-          <ListView
+          <FlatList
             style={styles.listViewContainer}
-            dataSource={dataSource}
-            renderRow={(rowData, sectionID, rowID) => this.renderItem(rowData, rowID, castButton, castButtonActive)}
+            data={devices}
+            extraData={this.state}
```

Comment:
`FlatList` checks `extraData` changes to rerender, but we also have `opacity` in the state. I'm not sure, but probably there is a way to pass there only `selectedID`, but we need to check if it will rerender. Otherwise it rerenders every time opacity changes too.

Reply:
done